components/atom/progressBar/src/ProgressBarCircle/Circle/index.js

```diff
@@ -88,6 +90,10 @@ Circle.propTypes = {
   strokeWidth: PropTypes.node.isRequired,
   /** boolean to activate/desactivate animations */
   withAnimation: PropTypes.bool,
+  /** When progress stroke is bigger than main one, it would be double in width  */
+  outerStrokeWidth: PropTypes.bool,
```

Comment:
a boolean prop should have a boolean-ish name IMHO...

how about `hasOuterStrokeDoubleWidth`?